README.md

```diff
@@ -1 +1,81 @@
-# countries-infomation
\ No newline at end of file
+![](https://img.shields.io/badge/Terbeche-Mostefa-violet)
+
+# countries-infomation
+
+> In this project, I have worked with the real live data from the RESTCountries API. This web application provides some information about more than 250 countries worldwide like the area, population, capital, continent, and the official flag.
+
+## Screen shots
+
+- ![](./screen/screen1.PNG)
+- ![](./screen/screen2.PNG)
+- ![](./screen/screen3.PNG)
+
+## Built With
+
+- ![](https://img.shields.io/badge/Github-blueviolet)
+- ![](https://img.shields.io/badge/Javascript-blue)
+- ![](https://img.shields.io/badge/HTML-purple)
+- ![](https://img.shields.io/badge/CSS-blue)
+- ![](https://img.shields.io/badge/WEBPACK-violet)
+- ![](https://img.shields.io/badge/Babel-violet)
+- ![](https://img.shields.io/badge/API-black)
+- ![](https://img.shields.io/badge/JSON-violet)
+- ![](https://img.shields.io/badge/REACT-blue)
+- ![](https://img.shields.io/badge/REDUX-blue)
+
+## Live Demo (if available)
+
+[Live Demo Link](https://countries-information.herokuapp.com/)
+
+## Author
+
+👤 **Terbeche Mostefa**
+
+- GitHub: [@githubhandle](https://github.com/Terbeche)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mustapha-terbeche/)
+
+## Setup
+
+If you wish to run the project locally, please do the following:
+
+1.- Clone the repository to your local machine. - **`git clone https://github.com/Terbeche/countries-infomation.git`** - **`cd spacex`**
+
+2.- Install Node.js and npm if you haven't already.
+To install Node.js, follow the instructions on [Node.js](https://nodejs.org/en/).
+
+To install npm with the necessary dependencies, run the following command in your terminal:
+
+```bash
+npm install
+```
+
+3.- Once you have the setup and the project locally, you can run the project using the following command in your terminal:
+
+```bash
+npm run build
+```
+
+4.- and if you want to watch the changes live, run the following command (it will run the website on localhost:3000):
+
+```bash
+npm start
+```
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Terbeche/countries-infomation/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- This project was based on the **RESTCountries API**
+- Microverse
```

Comment:
- [ ] The "Acknowledgements" section should give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).